client/src/views/Chat.vue

```diff
@@ -289,7 +313,19 @@ export default {
   mounted() {
     // socket to recieve messages
     socket.on("receive_message", (data) => {
-      const message = this.privateKey.decrypt(data.message);
+      const encryptedKey = data.key;
+
+      //decrypts the symmetric key using their private key.
+      const key = this.privateKey.decrypt(encryptedKey);
+      const iv = data.iv;
+      //decrypts the message using the symmetric key.
+      const decipher = forge.cipher.createDecipher("AES-CBC", key);
+      decipher.start({ iv: iv });
+      decipher.update(data.message);
```

Comment:
This is the answer we are looking for.

https://github.com/digitalbazaar/forge/issues/315#issuecomment-152741165

```suggestion
      decipher.update(forge.util.createBuffer(data.message));
```

Reply:
ok, thanks for the review, I will make the changes